rstdocs/index.rst

```diff
@@ -174,6 +174,21 @@ using sets as parameters.
 Nested structures are also not directly allowed but you can use `json <https://www.edgedb.com/docs/edgeql/insert#bulk-inserts>`_
 instead.
 
+By default EdgeDB will ignore embedded structs when marshaling/unmarshaling.
+To treat an embedded struct's fields as part of the parent struct's fields
+tag the embedded struct with `edgedb:$inline`.
```

Comment:
```suggestion
tag the embedded struct with ``edgedb:$inline``.
```

Reply:
Good catch!

The `rst` files are generated from the go documentation comments. Go documentation comments don't have anything analogous to the double back ticks in `rst`. The back ticks here are literally what the user should put in their source `` `edgedb:"$inline"` ``. I guess the generator needs to be updated to escape the back tics.

Also looks like I forgot the quotes!